test/replication/qsync_basic.result

```diff
@@ -222,6 +225,9 @@ test_run:switch('replica')
  | ---
  | - true
  | ...
+test_run:wait_lsn('replica', 'default')
```

Comment:
Strictly speaking this `wait_lsn` isn't needed. The `replace{4}` is replicated synchronously, so by the time the command returns on master it must've already been replicated to the replia.

You may leave that wait_lsn here if you want to make sure that `CONFIRM` reaches the replica as well. I wouldn't bother though.

Reply:
agree, removed